spec/oppiabotSpec.js

```diff
@@ -9,9 +9,12 @@ const pullRequestpayload = require('../fixtures/pullRequestPayload.json');
 
 describe('Oppiabot\'s', () => {
   let robot;
+  /**
+   * @type {import('probot').Octokit} context
+   */
```

Comment:
This is undefined here, right? So why are we adding type? Also, I see this variable is storing dict

Reply:
Adding the type helps in mocking the new functions in the probot upgrade as the editor will help with better IntelliSense.

Updated to show `github` instead of `context`

Reply:
No, that's the correct type.